docs/troubleshooting.md

```diff
@@ -221,3 +239,22 @@ Agent, which can print the runtime information of `antrea-controller` and
 information on a Node, dump Antrea OVS flows, and perform OVS packet tracing.
 Refer to the [`antctl` guide](/docs/antctl.md#usage) to learn how to use these
 commands.
+
+## Profiling Antrea components
+
+The easiest way to profile the Antrea components is to use the Go
+[pprof](https://golang.org/pkg/net/http/pprof/) tool. Both the Antrea Agent and
+the Antrea Controller use the k8s apiserver library to server their API, and
```

Comment:
k8s -> K8s

Are you using "server their API" on purpose, or you meant "serve"?

Reply:
that was a typo, thanks